hydroflow_lang/src/graph/ops/multiset_delta.rs

```diff
@@ -5,30 +5,43 @@ use super::{
     WriteContextArgs, RANGE_0, RANGE_1,
 };
 
-// TODO(mingwei): more doc
-/// Multiset delta from the previous tick.
+/// The multiset inverse of [`persist()`](#persist).
+///
+/// > 1 input stream of `T`, 1 output stream of `T`, where `T: Eq + Hash`
+///
+/// For set semantics, [`unique()`](#unique) can be thought of as a "delta" operator, the inverse
+/// of [`persist()`](#persist). In `persist`, new items come in, and all items are repeatedly
+/// released out. Conversely, `unique` take repeated items in, and only releases the new ones out.
+///
+/// This operator does a similar inversion but for multiset semantics, with some caveats. When it
+/// receives duplicate items, instead of ignoring them, it "subtracts" them from the items received
+/// in the previous tick, and only releases new items if there are more than in the previous tick.
+/// However unlike `unique`, this count is only maintained for the previous tick, not over all time.
```

Comment:
Given the proposed rewording above, maybe rephrase to match.

I'm actually confused as to whether it remembers the MAX number of copies over time, or just the last tick number?  If it's MAX, then we should probably implement a generic `lattice_delta`, where this is a specific one for multiset lattices. If it's not MAX, then I'd like to discuss semantics --- it'd be weird for this to be the only 1-tick windowed operator (other than `next_tick`).

Reply:
It is only the previous tick. The semantics become weird if it's not somewhere after a `persist` operator